packages/sources/dxdao/src/endpoint/tvl.ts

```diff
@@ -0,0 +1,65 @@
+import { Requester, Validator, Logger } from '@chainlink/ea-bootstrap'
+import { Config, ExecuteWithConfig } from '@chainlink/types'
+import { getRpcLatestAnswer } from "@chainlink/ea-reference-data-reader"
+import { JSON_RPC_URL } from '../config'
+import { ethers, BigNumber } from 'ethers'
+
+export const NAME = 'TVL'
+
+const customParams = {
+  wethContractAddress: true,
```

Comment:
I had thought about setting this to something more generic but it looks like we may only be querying from the WETH token address to calculate TVL.  Interested to hear any opinions on this.